README.md

```diff
@@ -463,7 +462,10 @@ is tagged.
 
 1. Once the final release candidate has rolled out successfully without issue on the
    above staging infrastructure, the release manager tags 2 additional `stacks-blockchain`
-   team members to review the `develop -> master` PR.
+   team members to review the `develop -> master` PR. If there is a merge conflict in this
```

Comment:
is this new code?

Reply:
No -- this is just a merge from `master` to `develop`, so all these changes are already in `master`. The code changes are limited to the benchmark fixes.

Reply:
I think having docs changes does not count as new code.